src/Spectre.libGenetic/MutationOperator.h

```diff
@@ -35,13 +36,23 @@ class MutationOperator
     /// <param name="mutationRate">The mutation rate.</param>
     /// <param name="bitSwapRate">The bit swap rate, in the case of mutation.</param>
     /// <param name="rngSeed">The RNG seed.</param>
-    explicit MutationOperator(double mutationRate, double bitSwapRate, Seed rngSeed = 0, size_t minimalFillup=0, size_t maximalFillup=std::numeric_limits<size_t>::max());
+    /// <param name="minimalFillup">The maximal fillup.</param>
+    /// <param name="maximalFillup">The minimal fillup.</param>
+    /// <param name="individualFeasibilityCondition">The individual feasibility condition.</param>
+    explicit MutationOperator(double mutationRate, double bitSwapRate, Seed rngSeed = 0, size_t minimalFillup=0, size_t maximalFillup=std::numeric_limits<size_t>::max(),
```

Comment:
Similarly, as in the case of `Crossover`:

1) Having two options of calling this from the outside is highly confusing, especially that one is assumed to be never called outside.
2) Do you plan on removing `minimalFillup` and `maximalFillup` in this or in the next review?

Reply:
next review, done